src/mac/frame/security/default.rs

```diff
@@ -0,0 +1,53 @@
+//! Provides a default AEAD to satisfy the type requirements for (de-)serializing frames without
+//! any security
+
+use ccm::aead::generic_array::{
+    typenum::consts::{U1, U16},
+    GenericArray,
+};
+use cipher::{BlockCipher, NewBlockCipher};
+
+use super::{DeviceDescriptorLookup, KeyLookup};
+
+/// A struct that fullfills all of the type checks, but is not actually capable of
+/// performing any of the operations
+pub struct Unimplemented();
```

Comment:
```suggestion
pub struct Unimplemented;
```

Not a big deal, just looks slightly cleaner.